pkg/modules/testdata/cr_auth_proxy_v180.yaml

```diff
@@ -9,16 +9,16 @@ spec:
     - name: authorization-proxy-server
       # enable: Enable/Disable csm-authorization
       enabled: true
-      configVersion: v1.6.0
+      configVersion: v1.8.0
```

Comment:
There are env variables in this file that aren't present in authorization v1.8.0. Please align the test file with this samples file: https://github.com/dell/csm-operator/blob/main/samples/authorization/csm_authorization_proxy_server_v180.yaml